WordPressAuthenticator/Model/LoginFields.swift

```diff
@@ -165,7 +165,7 @@ public class LoginFieldsMeta: NSObject {
          requiredMultifactor: Bool = false,
          socialService: SocialServiceName? = nil,
          socialServiceIDToken: String? = nil,
-         googleUser: GIDGoogleUser? = nil,
+         googleUser: SocialService.User? = nil,
          appleUser: SocialService.User? = nil) {
```

Comment:
I wrote it elsewhere already, but it seems to me like these parameters are ripe for consolidation into a single type (`SocialUser`?).

I'm not keen on introducing such a change right now because I'd like to stay focused on removing the Google SDK only. The tradeoff is that we'll have to cycle through two major version changes, but that seems acceptable. It's just a number after all.